test/test.cpp

```diff
@@ -163,3 +173,25 @@ TEST(Clip, Points) {
     ASSERT_EQ(expected1, clipped1);
     ASSERT_EQ(expected2, clipped2);
 }
+
+TEST(GetTile, USStates) {
+    const auto geojson = mapbox::geojson::parse(loadFile("test/fixtures/us-states.json"));
+    GeoJSONVT index{ geojson.get<mapbox::geojson::feature_collection>() };
+
+    const auto features = index.getTile(7, 37, 48).features;
+    const auto expected = parseJSONTile(loadFile("test/fixtures/us-states-z7-37-48.json"));
+    ASSERT_EQ(features == expected, true);
```

Comment:
This may be due to the fact original `geojson-vt-cpp` converted all values to strings (kind of a stopgap). This branch should handle property values properly, so expected fixtures might need an update.